Test/GetPowerShellModule.Tests.ps1

```diff
@@ -174,7 +246,46 @@ Describe 'GetPowerShellModule.when called with clean mode' {
         Init
         GivenModule 'Rivet'
         GivenCleanMode
-        WhenPowershellModuleIsRan
-        ThenModuleShouldNotExist
+        WhenTaskRun
+        ThenModuleNotInstalled
     }
 }
+
+Describe 'GetPowerShellModule.when allowing prerelease versions' {
+    AfterEach { Reset }
+    It 'should install a prelease version' {
+        Init
+        GivenModule 'Whiskey'
+        GivenVersion '0.43.*-*'
+        GivenPrereleaseAllowed
+        WhenTaskRun
+        ThenModuleInstalled -AtVersion '0.43.0-beta1416'
+    }
+}
+
+Describe 'GetPowerShellModule.when installing to custom directory' {
+    AfterEach { Reset }
+    It 'should install to a custom directory' {
+        Init
+        GivenModule 'Zip'
+        GivenPath 'FubarSnafu'
+        WhenTaskRun
+        ThenModuleInstalled -InDirectory 'FubarSnafu'
+        GivenCleanMode
+        WhenTaskRun
+        ThenModuleNotInstalled -InDirectory 'FubarSnafu'
+    }
+}
+
+Describe 'GetPowerShellModule.when importing module after installation' {

```

Comment:
Add a test for what happens when importing a module that's already imported. Newer version? Older version?